Source/Scene/DracoLoader.js

```diff
@@ -286,6 +286,28 @@ DracoLoader.decodePointCloud = function (parameters) {
   ]);
 };
 
+/**
+ * Decodes a buffer view. Returns undefined if the task cannot be scheduled.
+ *
+ * @param {Object} options Object with the following properties:
+ * @param {Uint8Array} options.array The typed array containing the buffer view data.
+ * @param {Object} options.bufferView The glTF buffer view object.
+ * @param {Object.<String, Number>} options.compressedAttributes The compressed attributes.
+ * @param {Boolean} options.dequantizeInShader Whether POSITION and NORMAL attributes should be dequantized on the GPU.
+ *
+ * @returns {Promise} A promise that resolves to the decoded indices and attributes.
+ * @private
+ */
+DracoLoader.decodeBufferView = function (options) {
```

Comment:
Most of the code in this file can be removed once the new `Model` is ready. I needed to add this function temporarily. It conforms to the same input/output properties that `decodeDraco` already uses.